src/javascript/app_2/App/Components/Form/input_field.jsx

```diff
@@ -30,22 +30,22 @@ const InputField = ({
     const changeValue = (e) => {
         if (type === 'number') {
             const is_empty = !e.target.value || e.target.value === '';
-            const signed_regex = is_signed ? '[\\+-]?' : '';
+            const signed_regex = is_signed ? '(?!^([-]0|[-+]?)$)^[+-]?' : '^';
```

Comment:
@easteregg 
Would you plz change `(?!^([-]0|[-+]?)$)` to `(?!^[-+]0?)$)` since it shouldn't accept `+0` and `-0` as a valid number.

Reply:
The proposed regex change will accept the empty string as a valid number. please check this out: 

https://regex101.com/r/2zZLns/1

Reply:
> The proposed regex change will accept the empty string as a valid number. please check this out:
> 
> https://regex101.com/r/2zZLns/1

yes, it needs a `?` after `[-+]`. it should be `(?!^[-+]?0?)$)`